docs/funnel-stages.md

```diff
@@ -0,0 +1,144 @@
+
+Companies may move into an inactive stage when activity drops and does not resume over 60 or more days. If activity resumes, the company will return to the last active stage, and points will reset. 
+
+
+# Recommended Sort<a id="recommended-sort"></a>
+
+The **Recommended Sort** reorders the list of companies in the activity table in descending order, first by their point value, then by number of events, and finally by most recent to oldest last seen value. This sort option lets you view the companies most worthy of your attention first. 
+
+![](https://lh7-us.googleusercontent.com/sZcf97LVEvu42H9AX_53Ui1LNEu7qEljM8sqlPyH2mne7qt5tlFU-l92l_sRmt7hcQZpUrEgJFO2XVELuwxp-UQqQN3TWuDNw3B3qkWe-H16z3GTHCDpecuhBBGiPlqKwkuBvmWk6yLbGbRQQTBJvPQbzui7W1AgT72nJN_o88RSwKhz45vbwTc4T6mnGw)
+
+
+# Company Journey<a id="company-journey"></a>
+
+When selecting a company to view in greater detail from any of the Company analytics charts, Administrators will see greater detail within the **Company Journey**. The company’s current funnel stage will be displayed in the top left, next to STATUS. Hover over the graphical calendar in the Company Journey to see details of the views and downloads that occurred over time. Below the graphical calendar are aggregated views of the total activity within each stage, including the current stage. 
+
+![](https://lh7-us.googleusercontent.com/C3ryxolhLe1iUgImK2BEjtWObXT2XSfPFEuHO4ZSKp18HhzkEIdNhf4PAlgS78o53QKD3E4PeIWsvOKR8aYYlD5xAr-8UnRSZ7Y93f7NRUDVRkifHq3NAA7ghpYqNZ5_HwhPfGJDvYvbtvc-VkiTF21hqkS8Pgi8jj7MUSRVSYr41SnU4QnfYdVmDHbYmA)
+
+
+# Event Importance<a id="event-importance"></a>
+
+**Event Importance** allows you to determine the weight that certain events carry over others. For instance, a download should carry more weight than a page view on your public website as it shows higher interest. This importance can be defined when setting up a new pixel or package and can be edited anytime. The importance will default to medium. 
+
+You’ll see the Event Importance noted on the Events detail charts. 
+
+![](https://lh7-us.googleusercontent.com/oyWd-rh2tylqA5qKuCCcx8gs3At5xivn5yZqfEBYdVHKLRu3GdbTWjSo5vXCljjEjorPrybE7EYqr39tlmKqq9qds6bTG5xM3M5gJ20NEP7umVGSPYgDpX841QXf5_FoextqER7BiIy2u3RD_4XIzb12D-VGNYdsORcxkHaCLwupP1zf9m27sCaPZz3KYg)
+
+Use the Event Importance slider to set or edit the importance of a Package or Pixel. 
+
+![](https://lh7-us.googleusercontent.com/JQtgtt5nxqQjrmAKPMIU3FioTFYk80FujjpVP8VyWrU4wmKRRTS3M1Yc2a_kMkaJBXo33l5A-2jKERg8rGCDTlQX0LyjyTk0i8QjmaZoWizgw-hYrfhrIvM9dvkPmSraFG92zdvzv3KefkqIbnInArJ2IZcSKmJAXtnBLnSRkLgckR-o29L20CYCkvqdFg)
+
+
+# Using Your Scarf Funnel Stage Data<a id="using-your-scarf-funnel-stage-data"></a>
+
+Scarf's data combines lead scoring's best features with intent data. Real-time activity highlights the companies actively researching and testing your open-source software. This data facilitates operationalizing the usage and intent data provided by your Scarf Gateway. 
```

Comment:
We may want to add something about optimizing the funnel itself. If you are noticing drop off at certain stages, it may indicate onboarding improvements you can make. Maybe your docs need work, maybe product positioning is off, etc.